posthog/api/person.py

```diff
@@ -23,6 +24,15 @@ def get_last_event(self, person: Person) -> Union[dict, None]:
         else:
             return None
 
+    def get_first_event(self, person: Person) -> Union[dict, None]:
+        if not self.context['request'].GET.get('include_last_event'):
+            return None
+        first_event = Event.objects.filter(team_id=person.team_id, distinct_id__in=person.distinct_ids).order_by('timestamp').first()
```

Comment:
Due to the way we've implemented $identify calls in `capture.py` it's possible a Person exists without an event, which would case this line (and thus the request) to fail. Worth catching `Event.DoesNotExist`